platformcomponents/desktop/date-picker.json

```diff
@@ -0,0 +1,50 @@
+{
+  "datePicker": {
+    "comment": "Applied to date picker.",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=51360%3A117328",
+    "date":{
+      "background":{
+        "#normal": "@theme-button-secondary-normal",
+        "#hovered": "@theme-button-secondary-hover",
+        "#pressed": "@theme-button-secondary-pressed",
+        "#disabled": "@theme-button-secondary-normal",
+        "#active": "@theme-background-accent-normal"
+      },
+      "text": {
+        "primary": {
+            "#normal": "@theme-text-primary-normal",
+            "#disabled": "@theme-text-primary-disabled",
+            "#active": "@theme-common-text-white"
+        },
+        "secondary": {
+            "#normal": "@theme-text-secondary-normal"
+        }
+      },
+      "border": {
+        "#normal": "@theme-outline-button-normal"
+      },
+      "unreadBadge": {
+        "#normal": "@theme-background-accent-normal",
+        "#active": "@theme-common-text-white"
+      }
+    },
+    "day":{
+      "background":{
+        "#normal": "@theme-button-secondary-normal",
+        "#hovered": "@theme-button-secondary-hover",
+        "#pressed": "@theme-button-secondary-pressed",
+        "#disabled": "@theme-button-secondary-normal",
+        "#active": "@theme-background-accent-normal"
+      },
+      "text": {
+        "#normal": "@theme-text-primary-normal",
+        "#disabled": "@theme-text-primary-disabled",
```

Comment:
worth adding the text for #hover and #pressed here too, even if they are the same as in #normal